near-contract-standards/src/non_fungible_token/event.rs

```diff
@@ -0,0 +1,174 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "standard")]
+#[serde(rename_all = "snake_case")]
+pub enum NearEvent {
+    Nep171(Nep171Event),
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Nep171Event {
+    pub version: String,
+    #[serde(flatten)]
+    pub event_kind: Nep171EventKind,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "event", content = "data")]
+#[serde(rename_all = "snake_case")]
+#[allow(clippy::enum_variant_names)]
+pub enum Nep171EventKind {
+    NftMint(Vec<NftMintData>),
+    NftTransfer(Vec<NftTransferData>),
+    NftBurn(Vec<NftBurnData>),
+}
+
+#[skip_serializing_none]
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
Flip the order of these lines, so `derive` is before any use of the derived macro attributes

Reply:
I tried and the code failed to compile.